src/ui/frmpreferences.cpp

```diff
@@ -150,30 +150,28 @@ void frmPreferences::on_buttonBox_accepted()
     s.setValue("Extensions/Runtime_Nodejs", ui->txtNodejs->text());
     s.setValue("Extensions/Runtime_Npm", ui->txtNpm->text());
 
+    const Editor::Theme& newTheme = Editor::themeFromName( ui->cmbColorScheme->currentText() );
```

Comment:
You should use `ui->cmbColorScheme->currentData()` here instead of `currentText()`, like it was before. Same thing for some of the other changes in this file.
This is because the "display text" of a control is rarely reliable data and might change based e.g. on user locale.